proxy/src/serverless/backend.rs

```diff
@@ -422,8 +429,13 @@ impl ConnectMechanism for HyperMechanism {
 
         let pause = ctx.latency_timer_pause(crate::metrics::Waiting::Compute);
 
-        // let port = node_info.config.get_ports().first().unwrap_or_else(10432);
-        let res = connect_http2(&host, 10432, timeout).await;
+        let port = node_info
+            .config
+            .get_ports()
+            .first()
+            .copied()
+            .unwrap_or_else(10432);
```

Comment:
The change is fine, but I think that we should consider creating a separate file to put these constants (like 10432 port#( into some "config" module in the future. Also, I believe it would be useful to have this new file(s) as a storage for some tunables that can be fetched from some config: it can be config yaml or from fetched from the cluster mgmt tunables

Reply:
Thinking about this. I think I'd rather error in this event